src/Entity.php

```diff
@@ -545,24 +545,13 @@ public function isRecursive()
 
     public function post_getFromDB()
     {
-        // Copy config "strategy" fields in corresponding "id" field.
-        if (($this->fields['calendars_strategy'] ?? 0) < 0) {
-            $this->fields['calendars_id'] = $this->fields['calendars_strategy'];
-        }
-        if (($this->fields['changetemplates_strategy'] ?? 0) < 0) {
-            $this->fields['changetemplates_id'] = $this->fields['changetemplates_strategy'];
-        }
-        if (($this->fields['contracts_strategy_default'] ?? 0) < 0) {
-            $this->fields['contracts_id_default'] = $this->fields['contracts_strategy_default'];
-        }
-        if (($this->fields['entities_strategy_software'] ?? 0) < 0) {
-            $this->fields['entities_id_software'] = $this->fields['entities_strategy_software'];
-        }
-        if (($this->fields['problemtemplates_strategy'] ?? 0) < 0) {
-            $this->fields['problemtemplates_id'] = $this->fields['problemtemplates_strategy'];
-        }
-        if (($this->fields['tickettemplates_strategy'] ?? 0) < 0) {
-            $this->fields['tickettemplates_id'] = $this->fields['tickettemplates_strategy'];
+        // Copy config "strategy" fields in corresponding "id" field
+        // when "strategy" is < 0.
+        foreach ($this->fields as $field_key => $value) {
```

Comment:
`transfers_strategy` was not listed. Using a `preg_match` permit to any `*_strategy` field.